Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,139 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+* {
+    font-family: Arial, Helvetica, sans-serif;
+    padding: 0;
+    margin: 0;
+}
+
+h1 {
+    font-size: 2rem;
+    font-weight: 500;
+}
+
+header {
+    height: 10vh;
+    border-bottom: 5px solid #4f5153;
+    display: flex;
+    align-items: center;
+}
+
+header img {
+    margin: 1vh 3vh
+}
+
+section {
+    height: 50vh;
+    background-image: url('/Exercise_D_E/assets/images/flowershop.jpg');
+    background-size: cover;
+}
+
+section h1 {
+    position: relative;
+    width: 100%;
+    text-align: center;
+    color: white;
+    top: 50%;
+    transform: translateY(-50%);
+}
+
+section button {
+    display: block;
+    margin: 3vh auto 0 auto;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    padding: 1.5vh 1vh;
+    border-radius: 300px;
+    background-color: #7ed321;
+    border: transparent;
+    color: white;
+}
+
+footer {
+    height: 40vh;
+    background-color: #cfe7b5;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.footercontainer {
+    display: flex;
+    text-align: center;
+    justify-content: space-around;
+    align-items: center;
+    box-sizing: border-box;
+} 
+
+.footercontainer img {
+    width: 10vw;
+    height: auto;
+}
+
+.footercontainer p {
+    inline-size: 150px;
+    overflow-wrap: break-word;
+}
+
+@media (max-width: 390px) {
+    h1 {
+        font-size: 1.6rem;
+    }
+    h2 {
+        font-size: 1.2rem;
+    }
+    p {
+        font-size: 1rem;
+    }
+}
+
+@media (min-width: 767px) {
+    h1 {
+        font-size: 2.5rem;
+    }
+    h2 {
+        font-size: 2rem;
+    }
+    p {
+        font-size: 1.5rem;
+    }
+    .footercontainer p {
+        inline-size: 200px;
+        overflow-wrap: break-word;
+    }
+}
+
+@media (min-width: 910px) {
+    h1 {
+        font-size: 2.5rem;
+    }
+    h2 {
+        font-size: 2rem;
+    }
+    p {
+        font-size: 1.5rem;
+    }
+    .footercontainer p {
+        inline-size: 250px;
+        overflow-wrap: break-word;
+    }
+}
+
+@media (min-width: 1279px) {
+    h1 {
+        font-size: 2.5rem;
+    }
+    h2 {
+        font-size: 2rem;
+    }
+    p {
+        font-size: 1.5rem;
+    }
+    .footercontainer p {
+        inline-size: 300px;
+        overflow-wrap: break-word;
+    }
+}
```

Comment:
Great work with Media Queries thus making it responsive, but i would suggest the part where you wrote the footer to instead divide it into two parts and give each a flex display would be better as it would look more formatted on the Desktop